keyboards/smithrune/iron160/iron160_s/info.json

```diff
@@ -27,71 +27,71 @@
-                { "label": "*",     "matrix": [0, 8 ], "w": 1, "x": 8, "y": 0 },
-                { "label": "(",     "matrix": [0, 9 ], "w": 1, "x": 9, "y": 0 },
-                { "label": ")",     "matrix": [0, 10], "w": 1, "x": 10, "y": 0 },
-                { "label": "_",     "matrix": [0, 11], "w": 1, "x": 11, "y": 0 },
-                { "label": "+",     "matrix": [0, 12], "w": 1, "x": 12, "y": 0 },
-                { "label": "Bspc",  "matrix": [0, 13], "w": 1, "x": 13, "y": 0 },
-                { "label": "Bspc",  "matrix": [4, 13], "w": 1, "x": 14, "y": 0 },
-                { "label": "Tab",   "matrix": [1, 0 ], "w": 1.5, "x": 0, "y": 1 },
-                { "label": "Q",     "matrix": [1, 1 ], "w": 1, "x": 1.5, "y": 1 },
-                { "label": "W",     "matrix": [1, 2 ], "w": 1, "x": 2.5, "y": 1 },
-                { "label": "E",     "matrix": [1, 3 ], "w": 1, "x": 3.5, "y": 1 },
-                { "label": "R",     "matrix": [1, 4 ], "w": 1, "x": 4.5, "y": 1 },
-                { "label": "T",     "matrix": [1, 5 ], "w": 1, "x": 5.5, "y": 1 },
-                { "label": "Y",     "matrix": [1, 6 ], "w": 1, "x": 6.5, "y": 1 },
-                { "label": "U",     "matrix": [1, 7 ], "w": 1, "x": 7.5, "y": 1 },
-                { "label": "I",     "matrix": [1, 8 ], "w": 1, "x": 8.5, "y": 1 },
-                { "label": "O",     "matrix": [1, 9 ], "w": 1, "x": 9.5, "y": 1 },
-                { "label": "P",     "matrix": [1, 10], "w": 1, "x": 10.5, "y": 1 },
-                { "label": "]",     "matrix": [1, 11], "w": 1, "x": 11.5, "y": 1 },
-                { "label": "[",     "matrix": [1, 12], "w": 1, "x": 12.5, "y": 1 },
-                { "label": "|",     "matrix": [1, 13], "w": 1.5, "x": 13.5, "y": 1 },
-                { "label": "Caps",  "matrix": [2, 0 ], "w": 1.75, "x": 0, "y": 2 },
-                { "label": "A",     "matrix": [2, 1 ], "w": 1, "x": 1.75, "y": 2 },
-                { "label": "S",     "matrix": [2, 2 ], "w": 1, "x": 2.75, "y": 2 },
-                { "label": "D",     "matrix": [2, 3 ], "w": 1, "x": 3.75, "y": 2 },
-                { "label": "F",     "matrix": [2, 4 ], "w": 1, "x": 4.75, "y": 2 },
-                { "label": "G",     "matrix": [2, 5 ], "w": 1, "x": 5.75, "y": 2 },
-                { "label": "H",     "matrix": [2, 6 ], "w": 1, "x": 6.75, "y": 2 },
-                { "label": "J",     "matrix": [2, 7 ], "w": 1, "x": 7.75, "y": 2 },
-                { "label": "K",     "matrix": [2, 8 ], "w": 1, "x": 8.75, "y": 2 },
-                { "label": "L",     "matrix": [2, 9 ], "w": 1, "x": 9.75, "y": 2 },
-                { "label": ":",     "matrix": [2, 10], "w": 1, "x": 10.75, "y": 2 },
-                { "label": "\"",    "matrix": [2, 11], "w": 1, "x": 11.75, "y": 2 },
-                { "label": "~",     "matrix": [2, 12], "w": 1, "x": 12.75, "y": 2 },
+                { "label": "~",     "matrix": [0, 0 ], "w": 1,    "x": 0,     "y": 0 },
```

Comment:
```suggestion
                { "label": "~",     "matrix": [0, 0 ], "x": 0,     "y": 0 },
```
etc...

Reply:
I'm not sure I understand what I see here, you're recommending I ommit `"w": 1` ?

Reply:
Yes, the default key width and height is 1u so there is no need for it.

Reply:
Committed in 7f166e653e .